zebra-consensus/src/verify/block.rs

```diff
@@ -471,4 +496,80 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    #[spandoc::spandoc]
+    async fn coinbase() -> Result<(), Report> {
+        install_tracing();
+
+        // Service variables
+        let state_service = Box::new(zebra_state::in_memory::init());
+        let mut block_verifier = super::init(state_service);
+
+        // Get a header of a block, can call function if #558 is merged
```

Comment:
I merged #558.

Reply:
I had a hard time trying to use the test utility calls like `block_header()` in consensus. By this ended up adding a vector with a header to `zebra_chain` that can be used everywhere. This was done in a separated commit https://github.com/ZcashFoundation/zebra/pull/593/commits/0cbe67c99be411d0dcfd143edf2c1658c8b50753 as it can be a bit unrelated.